confgenerator/confgenerator.go

```diff
@@ -147,18 +150,19 @@ func generateFluentBitConfigs(logging *logging, logsDir string, stateDir string)
 	fbTails := defaultTails(logsDir, stateDir)
 	fbStackdrivers := defaultStackdriverOutputs()
 	fbSyslogs := []*conf.Syslog{}
+	fbWinlogs := []*conf.WindowsEventlog{}
 	fbFilterParsers := []*conf.FilterParser{}
 	fbFilterAddLogNames := []*conf.FilterModifyAddLogName{}
 	fbFilterRewriteTags := []*conf.FilterRewriteTag{}
 	fbFilterRemoveLogNames := []*conf.FilterModifyRemoveLogName{}
 
 	if logging.Service != nil {
-		fileReceiverFactories, syslogReceiverFactories, err := extractReceiverFactories(logging.Receivers)
+		fileReceiverFactories, syslogReceiverFactories, winlogReceiverFactories, err := extractReceiverFactories(logging.Receivers)
```

Comment:
Since there are also windows file based logs, can we include "event" in the naming to help distinguish them? e.g. "winevent"

Reply:
Done